spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/RabbitServiceConfiguration.java

```diff
@@ -34,11 +35,12 @@
  * @author Mark Fisher
  * @author Dave Syer
  * @author Glenn Renfro
+ * @author David Turanski
  */
 @Configuration
 @ConditionalOnClass(RabbitMessageChannelBinder.class)
 @ConditionalOnMissingBean(RabbitMessageChannelBinder.class)
-@ImportResource("classpath*:/META-INF/spring-cloud-stream/binder/rabbit-binder.xml")
+@Import(RabbitMessageChannelBinderConfiguration.class)
 @PropertySource("classpath:/META-INF/spring-cloud-stream/rabbit-binder.properties")
```

Comment:
Just as a clarification: I meant it in what concerns the project structure. We can definitely create a centralized configuration for use with spring-cloud-config for example.